lib/pages/movie_details.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:movie_card/widgets/movie/movie_presentation.dart';
+import 'package:movie_card/widgets/movie/overview.dart';
+import 'package:movie_card/widgets/movie/movie_details_custom_nav_bar.dart';
+import 'package:movie_card/widgets/movie/cast.dart';
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({super.key, required this.title});
+
+  final String title;
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  static const double padFinal = 130;
+  static const String title = "The Super Mario Bros. Movie";
+  static const String date = "2023-04-05";
+  static const String overview =
+      "While working underground to fix a water main, Brooklyn plumbers—and brothers—Mario and Luigi are transported down a mysterious pipe and wander into a magical new world. But when the brothers are separated, Mario embarks on an epic quest to find Luigi.";
+  static const images = [
+    "assets/cast/1.jpg",
+    "assets/cast/2.jpg",
+    "assets/cast/3.jpg",
+    "assets/cast/4.jpg",
+    "assets/cast/5.jpg",
+    "assets/cast/6.jpg",
+  ];
+  static const genres = "Animation, Family, Adventure, Fantasy, Comedy";
+  static const double percentIndicator = 0.78;
+  static const posterName = "MarioBros_poster";
+  static const backdropName = "MarioBros_backdrop";
```

Comment:
All the information related to the movie, could be move it to a factory constructor on a Movie class